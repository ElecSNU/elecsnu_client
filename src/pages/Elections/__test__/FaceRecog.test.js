import { render, screen, fireEvent, cleanup } from '@testing-library/react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
// import Voting from './../Elections'
import { toast } from 'react-toastify';
import React, { useEffect, useState } from 'react';
// import Face from './../FaceRecog';
// import FaceRecog from './../FaceRecog';
// import ParticleBackground from '../../components/ParticleBackground/ParticleBackground';


test('Tests the rendering of Face recognition component in all its entirety', () => {
    const div = document.createElement("div")
    const tree = renderer.create(<index/>).toJSON();
    expect(tree).toMatchSnapshot(); 
   });