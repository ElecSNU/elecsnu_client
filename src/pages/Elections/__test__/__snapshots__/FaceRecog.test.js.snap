// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the rendering of Face recognition component in all its entirety 1`] = `<index />`;
